README.md

```diff
@@ -4,3 +4,52 @@ pol.is an AI powered sentiment gathering platform. More organic than surveys, le
 If you don't want to deploy your own instance of Polis, you can use our SaaS version [here](https://pol.is/home)
 Polis [can be easily embedded](http://docs.pol.is/usage/Embedding.html) on your page as an iframe.
 
+## Installation
+
+### Dependencies
+
+* PostgreSql `(>= 9.5.4.1)`
+* Node `6.11.1`
+We recommend installing [nvm](https://github.com/creationix/nvm) so you can easily switch between your favorite flavors of node.
+* NPM `3.3.8`
+
+### Setup
+
+1. Create a new datbase. You can name it whatever you please.
+    ```sh
+    create database polis;
+    ```
+1. Connect to the new database then run `postgres/db_setup_draft.sql` in its shell
+    ```
+    \connect polis;
+    \i postgres/db_setup_draft.sql`;
+    ```
+1. Create development envs file
+    ```sh
+    $ cp .env_dev_local_db_template .env_dev
+    ```
+1. Update database connection settings in `.env_dev`. Replace the username, password, and database_name in the DATABASE_URL
+    ```
+    export DATABASE_URL=postgres://your_pg_username:your_pg_password@localhost:5432/your_pg_database_name
```

Comment:
Might be worth helping people find their port, as it can vary when there are multiple versions of postgres available in the package repo for your system. For example. mine is `5434`.

But tmaybe that's out of scope, which is totally fair :)

Reply:
@patcon Feel free to add a quick pointer about this!